vector/upload_new_vector_version.sh

```diff
@@ -1,25 +1,34 @@
 #!/usr/bin/env bash
 
 VERSION=${1:?"Missing version number argument (arg 1)"}
-NEXUS_USER=${2:?"Missing Nexus username argument (arg 2)"}
+RELEASE=${2:?"Missing release number argument (arg 2)"}
+NEXUS_USER=${3:?"Missing Nexus username argument (arg 3)"}
+
+ARCHITECTURES=(
+    x86_64
+    aarch64
+)
```

Comment:
@Maleware should we start to document stuff like this for multi arch? In case there are any additions...

Reply:
Yes we should somewhere. Does it makes sense to add a multi-arch section in our docs?